bedrock/mozorg/templates/mozorg/contribute/tasks/devtools-challenger.html

```diff
@@ -24,15 +24,15 @@
       </header>
       <div class="step-content">
         <img src="{{ static('img/contribute/signup/devtools.png') }}" width="620" class="feature-img" alt="" />
-        <a href="http://devtoolschallenger.com/" rel="external" class="button-blue action" data-action="challenger" data-task="devtools" data-step="one" data-complete="true" target="_blank">
+        <a href="http://devtoolschallenger.com/" rel="external" class="button-blue action" data-action="challenger" data-task="devtools" data-step="one" data-complete="true" target="_blank" rel="noopener noreferrer">
           {% if l10n_has_tag('contribute-tasks-0516') %}
             {{ _('Visit Dev Tools Challenger') }}
           {% else %}
             {{ _('Learn More') }}
           {% endif %}
         </a>
         <h4>{{ _('Get involved') }}</h4>
-        {{ _('<a href="%s" id="get-involved" target="_blank">Join the Webdev Contribute Group</a>  and start collaborating with other Mozillians around the world.')|format('https://wiki.mozilla.org/Contribute/Webdev') }}
+        {{ _('<a href="%s" id="get-involved" target="_blank" rel="noopener noreferrer">Join the Webdev Contribute Group</a>  and start collaborating with other Mozillians around the world.')|format('https://wiki.mozilla.org/Contribute/Webdev') }}
```

Comment:
As `target="_blank"` is now considered both a [security risk](https://mathiasbynens.github.io/rel-noopener/) and a [performance issue](https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/), I think it is worth adding these.

@RCMainak - for situations like this you can use an l10n conditional tag, like this:

```
{ set url = 'https://wiki.mozilla.org/Contribute/Webdev' %}
{% if l10n_has_tag('target-links-noopener') %}
  {{ _('<a href="%s" id="get-involved" target="_blank" rel="noopener noreferrer">Join the Webdev Contribute Group</a>  and start collaborating with other Mozillians around the world.')|format(url) }}
{% else %}
  {{ _('<a href="%s" id="get-involved" target="_blank">Join the Webdev Contribute Group</a>  and start collaborating with other Mozillians around the world.')|format(url) }}
{% endif %}
```